x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/logic/indices_api/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { IndicesAPILogic } from './indices_api_logic';
```

Comment:
suggestion: remove this index.ts. Barrel files are a drag on compiler and type checking performance, and don't really add that much in a monolith.